js/send-mail.js

```diff
@@ -151,14 +151,16 @@ $(function () {
 
 	// cc/bcc toggling
 	$(document).on('click', '#new-message-cc-bcc-toggle', function() {
-		$('#new-message-cc-bcc').slideToggle();
-		$('#new-message-cc-bcc #cc').focus();
+		var newMessageCcBcc = $('#new-message-cc-bcc');
+		newMessageCcBcc.slideToggle();
+		newMessageCcBcc.find('#cc').focus();
 		$('#new-message-cc-bcc-toggle').fadeOut();
 	});
 
 	$(document).on('click', '#reply-message-cc-bcc-toggle', function() {
-		$('#reply-message-cc-bcc').slideToggle();
-		$('#reply-message-cc-bcc #cc').focus();
+		var replyMessageCcBcc = $('#reply-message-cc-bcc');
+		replyMessageCcBcc.slideToggle();
+		replyMessageCcBcc.find('#cc').focus();
```

Comment:
Is this replacement really the same? I may be wrong, but to me it looks like it was a selection if any element with classes ``reply-message-cc-bcc`` and ``cc`` before.
Your changes select any element of class ``reply-message-cc-bcc``, but then ``find`` searches any descendants of that selection, see https://api.jquery.com/find/